packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -351,9 +351,7 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
 
   DismissDirection get _dismissDirection => _extentToDirection(_dragExtent);
 
-  bool get _isActive {
-    return _dragUnderway || _moveController.isAnimating;
-  }
+  double get _dismissThreshold => widget.dismissThresholds[_dismissDirection] ?? _kDismissThreshold;
```

Comment:
I added a `_dismissThreshold` getter, since `widget.dismissThresholds[_dismissDirection] ?? _kDismissThreshold` was used in 4 different spots.